### PR TITLE
ENG-1512: XSS protection using content security policies

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/main/java/org/entando/entando/aps/tags/CspNonceTag.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/entando/aps/tags/CspNonceTag.java:36]: is EVAL_BODY_INCLUDE the right return value for currentToken == null?

File: src/main/java/org/entando/entando/aps/tags/CspNonceTag.java
Comment (w-caffiero-entando) [inline src/main/java/org/entando/entando/aps/tags/CspNonceTag.java:38]: @eugeniosant not sure to get this part, `out` internally already discriminate between escaped and unescaped  contents.

Comment (eugeniosant) [reply src/main/java/org/entando/entando/aps/tags/CspNonceTag.java]: @w-caffiero-entando I've removed this check

File: src/main/java/org/entando/entando/aps/tags/CspNonceTag.java
Comment (eugeniosant) [reply src/main/java/org/entando/entando/aps/tags/CspNonceTag.java:36]: @w-caffiero-entando If the token is null, nothing should happen. I changed the return value to the default result of doStartTag (SKIP_BODY)

**Review by w-caffiero-entando — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/87#pullrequestreview-540504477" title="Approved">w-caffiero-entando</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by w-caffiero-entando — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/87#pullrequestreview-540504585" title="Approved">w-caffiero-entando</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

Approval requirements bypassed by manually added approval.

This pull-request has been approved by: *<a href="https://github.com/entando/entando-engine/pull/87#pullrequestreview-540504585" title="Approved">w-caffiero-entando</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-engine).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-engine/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
